Resources/Private/ExtensionArtifacts/src/Hook/ExtensionInstallation.php

```diff
@@ -130,7 +130,7 @@ private function safeCopy($fullSourcePath, $fullTargetPath, $relativeWebDir = ''
 
     protected static function isTypo3CmsBinary($fullTargetPath)
     {
-        return strpos(file_get_contents($fullTargetPath), 'typo3-console.php') !== false;
+        return strpos(file_get_contents($fullTargetPath), 'typo3cms.php') !== false;
```

Comment:
that won't work, as the new file is called typo3-console.php

so it needs be be checked for both strings

Reply:
Well the new file does not need to be replaced at all, doesn't it?

Reply:
not now, but later maybe